test-result-collection-tool/src/main/assemblies/readme.txt

```diff
@@ -112,16 +112,18 @@ Some server-side errors can cause test result push failure even when the
 pushed XML is well formatted. You can use the skip-errors flag (-e option) to 
 force pushing such a test result.
 
-Password/Client secret handling
+Password handling
 -----------------
 
-For authentication, the user can use a username and password or generate an API access key as described here:
-https://admhelp.microfocus.com/octane/en/latest/Online/Content/AdminGuide/how_setup_APIaccess.htm
-The password/Client secret can be entered in the following ways:
+For authentication the user can use a username and password, or generate an API access key comprised of a client ID and client secret,
+as described here: https://admhelp.microfocus.com/octane/en/latest/Online/Help_Center.htm#CSHID=api_access_tab.
+If using an API access key: enter the client ID as the username, and the client secret as the password.
```

Comment:
we need to clarify that the recommended auth method is using API access key and not user & password. 

that for testing purposes it's possible to use user & password, but for production deployment only api access should be used.

Please discuss with docs how best to phrase this.

instead of 

> if using API access ....

should say

> When using API access ....